conf.toml

```diff
@@ -43,3 +43,7 @@ HostOrchestratorPort = 1081
 
 [WebRTC]
 STUNServers = ["stun:stun.l.google.com:19302"]
+
+# Regex patterns of allowed origins
+# e.g. "https://localhost:[0-9]+"
+CorsAllowedOrigins = []
```

Comment:
Let's use `CORSAllowedOrigins` instead.  https://google.github.io/styleguide/go/decisions.html#initialisms

Reply:
Changed to CORSAllowedOrigins.